web/client/src/components/infrastructure/elementTypes.ts

```diff
@@ -0,0 +1,23 @@
+export const elementTypeLabels = {
+	'bumper': 'Bumper',
+	'border': 'Border',
+	'track_end': 'Track End',
+	'simple_switch': 'Switch',
+	'as': 'Approach Signal',
+	'ms': 'Main Signal',
+	'ps': 'Protection Signal',
+	'eotd': 'End of Train Detector',
+	'spl': 'Speed Limit',
+	'tunnel': 'Tunnel',
+	'hlt': 'Halt',
+	'rtcp': 'RTCP',
+	'km_jump': 'KM Jump',
+	'line_switch': 'Line Switch',
+	'slope': 'Slope',
+	'cross': 'Cross',
+	'ctc': 'CTC',
+	'station': 'Station'
+};
```

Comment:
Maybe use an enum. For better readablitily

Reply:
I do not directly see how an enum would benefit this (right now). The keys here might be transferred from the server and used in the sent infrastructure (if the server would actually serve whole infrastructures like it used to).